how-to/use-theming/client/src/defaultPalettes.ts

```diff
@@ -0,0 +1,63 @@
+import type { CustomPaletteSet } from "@openfin/workspace/common/src/api/theming";
+
```

Comment:
the naming of the file seems to break with our naming convention. Shouldn't it be default-palettes.ts ?

Reply:
renamed